web/src/report/components/report-chart/report-chart.component.ts

```diff
@@ -27,27 +29,46 @@ export class ReportChart extends LitElement {
   @Inject('report')
   private reportService: ReportService;
 
+  @Inject('resource')
+  private resourceService: ResourceService;
+
   reportsCharts: Array<ReportChartResponse> = [];
 
+  accessChart: ReportChartResponse;
+
   membershipTrendsData: ReportChartResponse;
 
+  membershipAccessTrendsData: ReportChartResponse;
+
   membershipDistributionData: Array<ReportChartResponse> = [];
 
   membershipTrendsChart: GoogleChart;
 
+  membershipAccessTrendsChart: GoogleChart;
+
   membershipDistributionChart: GoogleChart;
 
+  resourceName = 'Bamity';
```

Comment:
It's strange we are setting a default resource manually. I would prefer if we grab the first one in the response rather than hard coding one.